### ISSUE TITLE
Voice Based Dashboard for Covid-19 in India and World

### ISSUE THREAD

**Issue by akashmitra** [akashmitra/callforcode#6]
**Problem**: People are having difficulty browsing through various sites and know about the current status on covid. And other urgent FAQs. 

**Solution**: Chatbot or Voice based application to update people on current statistics on covid. We can also answer FAQs as laid down by WHO and CDC.

**APIs**: 
https://api.covid19india.org/ 
https://covid19api.com/

**Techs**:
IBM Watson,
Nodejs,
IBM Discovery, (Optional)
Node Red (Optional)

**Comment by sas92** [comment]
This will also be a good idea. Because I personally feel the pain of visiting websites every now and then to get info.
A chatbot will be easy to create for this story. We can also broadcast recent updates periodically to the subscribers (only if the device supports).

**Comment by akashmitra** [comment]
Exactly. Opening up sites. Keeping tab of stuffs. One place voice based updates could be helpful. Plus basic information like nearest medicine shops. Infact we can merge this with issue#1.

**Comment by sas92** [comment]
Yes better to merge #1 to it. Without the stock/inventory part. Because it will be too much time taking to create an inventory management system.

**Comment by sas92** [comment]
We can also add a feature - Symptom checker.
Will ask users if facing symptoms from this list and suggest to visit doctor
```
Common symptoms include:
- fever
- tiredness
- dry cough

Other symptoms include:
- shortness of breath
- aches and pains
- sore throat
- and very few people will report diarrhoea, nausea or a runny nose.
```
Source - https://www.who.int/health-topics/coronavirus#tab=tab_3

**Comment by akashmitra** [comment]
Yeah. Something like this. But we'll have tons of other features, this being a minute one. 

Also, lets not get into inventory management, but what we can do is, geotag shops and flag then open\close based on user data. Interpolate on daily. But this is an added functionality. We can skip it for the MVP.
